### ISSUE TITLE
Install errors - on RStudio cloud

### ISSUE THREAD

**Issue by dvanic** [Sydney-Informatics-Hub/dataharvester#1]
```r
> initialise_harvester(earthengine = FALSE)
• Checkking python/conda install...
✔ Conda binary: /cloud/home/r199168/.local/share/r-miniconda/bin/conda
• Verifying python configuration...
✔ Using conda environment 'r-reticulate'
> # note: will need to restart dataharvester more than once
> # a third check will also happen to finalise dependencies
> initialise_harvester(earthengine = TRUE)
• Checkking python/conda install...
✔ Conda binary: /cloud/home/r199168/.local/share/r-miniconda/bin/conda
• Verifying python configuration...
✔ Using conda environment 'r-reticulate'
• Starting Earth Engine authetication...
Matplotlib is building the font cache; this may take a moment.
Error in py_module_import(module, convert = convert) : 
  ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /cloud/home/r199168/.local/share/r-miniconda/envs/r-reticulate/lib/python3.8/site-packages/rasterio/../../../libgdal.so.31)
> # note: will need to restart dataharvester more than once
> # a third check will also happen to finalise dependencies
> initialise_harvester(earthengine = TRUE)
• Checkking python/conda install...
✔ Conda binary: /cloud/home/r199168/.local/share/r-miniconda/bin/conda
• Verifying python configuration...
✔ Using conda environment 'r-reticulate'
• Starting Earth Engine authetication...
Error in py_module_import(module, convert = convert) : 
  AttributeError: partially initialized module 'rasterio' has no attribute '_loading' (most likely due to a circular import)
```

**Comment by marius-mather** [comment]
I think this is related, and is the reason `rasterio` fails to load/initialize properly:

```
• Starting Earth Engine authetication...
Error in py_module_import(module, convert = convert) : 
  ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /cloud/home/r1650462/.local/share/r-miniconda/envs/r-reticulate/lib/python3.8/site-packages/rasterio/../../../libgdal.so.31)
```

**Comment by sebhaan** [comment]
Error in py_module_import(module, convert = convert) : 
  ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /cloud/home/r1650365/.local/share/r-miniconda/envs/r-reticulate/lib/python3.8/site-packages/rasterio/../../../libgdal.so.31)

**Comment by januarharianto** [comment]
Fixed in 7ceb67393b5aed888eb63c58c1f94533f2d256b5
